app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -105,17 +108,28 @@ private fun HomeSectionHeaderContent(
         )
 
         onShowAllClick?.let {
-            ClickableText(
-                text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-                modifier = Modifier.padding(start = 16.dp)
-                    .semantics {
-                        contentDescription = description
+            TextButton(
+                text = stringResource(id = R.string.recent_tabs_show_all),
+                shouldUppercaseText = false,
+                buttonColors = textButtonColors(
+                    contentColor = if (isSystemInDarkTheme()) {
+                        PhotonColors.White
+                    } else {
+                        PhotonColors.Black
                     },
```

Comment:
this parameter is covered by `textColor = showAllTextColor`. We need to use `showAllTextColor` to adapt the text to a different color based on the selected wallpaper.

Reply:
This is to used add the ripple effect, I'll update the documentation to reflect this.